packages/ui-rating/src/RatingIcon/index.tsx

```diff
@@ -58,23 +58,20 @@ class RatingIcon extends Component<RatingIconProps, RatingIconState> {
 
   ref: Element | null = null
```

Comment:
Actually we can tell that it is HTMLSpanElement:
```suggestion
  ref: HTMLSpanElement | null = null
```